src/trumbowyg.js

```diff
@@ -1054,6 +1058,14 @@ Object.defineProperty(jQuery.trumbowyg, 'defaultOptions', {
             }
         },
 
+        // Fullscreen Mode management
+        fullscreenMode: function(isFullScreen){
+            var t = this;
+            // Check if fullscreen mode.
+            if(isFullScreen){
+                t.fullscreen();
```

Comment:
This can't work as is :/

fullscreen method toggle the fullscreen state, so the actual behaviour is weird.

Reply:
@Alex-D Could you elaborate on the weird behavior? I tried this in a project of mine with fullscreen option enabled by default with a fullscreen button as well. Worked for me. If you could elaborate, I dont mind going back and making changes.

Reply:
@Alex-D Thanks for the insight. Ill go back to the drawing board.

Reply:
is same and functional
 
el.trumbowyg('execCmd', {cmd: 'fullscreen'})